cli/src/lib/paths.js

```diff
@@ -109,7 +109,7 @@ module.exports = (cwd = process.cwd(), { typeScript } = {}) => {
             './.d2/shell/src/plugin.index.jsx'
         ),
         shellPluginEntrypoint: path.join(base, './.d2/shell/src/Plugin.jsx'),
-        // TODO: look at what to do with service-worker files whether to move to TS or not
+
```

Comment:
removed this todo - it's a general question about whether we should move this whole repo to TypeScript, not the place for it in this PR